src/index.jsx

```diff
@@ -1,10 +1,11 @@
+import React from 'react'
 import core from './core'
 import TailSpin from './TailSpin'
 
 export default (
   ComposedComponent,
   {
-    LoadingIndicator = TailSpin,
-    ...rest,
+    renderLoadingIndicator = (() => <TailSpin />),
```

Comment:
I chose to do that purposely in order to avoid the prop warnings... I didn't see the need for the props to use it as the spinner is pretty responsive inside its parent as is, and if you need more specific behavior you could simply replace the `renderLoadingIndicator` and supply the props you need.

That said, I do like the idea of passing `className` and `style` as mentioned in the issue.